### DISCUSSION TITLE
Should we consider an official Matrix room?

### DISCUSSION THREAD

**Discussion by ghost** [olive-editor/olive#2000] [Q&A]
or alternatively a Matrix room bridged with discord.

**Comment by elsandosgrande** [comment]
As far as I can tell, Martix works more like a group chat as opposed to an IRC server with multiple channels, which is more Discord's model. Since being able to keep the various topics discussed in their own categories/channels is necessary, Matrix doesn't seem like a good fit at all. I've of some open-source Discord clone out there, but the question is how far along it is. 

### Edit
Experimental unofficial Matrix space: https://matrix.to/#/#olive-video-editor:matrix.org

**Reply by elsandosgrande** [reply]
@uranuspucksaxophone Sorry, I wasn't aware of the new feature, as I last used Element last year 😅.

So, I've set up an unofficial space, but I can't say whether or not it will last; that depends on what happens in the future. Here's the invite link (even though it's a public space): https://matrix.to/#/#olive-video-editor:matrix.org. Also, bridging would be nice, but that would require talking with the other administrators on the server to add a bridge bot, not to mention more time to set up, so it's probably not gonna happen for a week or two at least.

**Reply by elsandosgrande** [reply]
@uranuspucksaxophone Matthew hasn't expressed any interest in dealing with anything more than the Discord server, but you can give it a shot and see what happens.
